ci/setup-template.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env sh
```

Comment:
Check this (and other shell scripts) with https://www.shellcheck.net/ and see if any valid warnings appear.

Reply:
Yes fixed